### ISSUE TITLE
Dropzone label should be required

### ISSUE THREAD

**Issue by alex-page** [Shopify/polaris-react#1588]
## Issue summary

Currently labels are optional for the DropZone. For accessibility this is not recommended.

## Expected behavior

The DropZone label is required with the option to visually hide it.

## Actual behavior

DropZone label is optional and can cause accessibility issues with an input and no label.

**Comment by dpersing** [comment]
Hi @alex-page! Do you want to tackle this as part of the work on #1565? Do you have enough info to go on, in that case?

**Comment by alex-page** [comment]
@dpersing This will be a breaking change and need to be in version 4!

**Comment by dpersing** [comment]
Closed with https://github.com/Shopify/polaris-react/pull/1565.
